composer.json

```diff
@@ -117,6 +116,7 @@
     "wpackagist-plugin/regenerate-thumbnails": "3.1.5",
     "wpackagist-plugin/user-switching": "1.6.0",
     "wpackagist-plugin/wordpress-seo": "19.3",
+    "wpackagist-plugin/wp-mail-smtp": "^3.5",
```

Comment:
@tarecord I know you didn't ask for my review, but you should add the specific version for plugins so during deploys a new version doesn't just get randomly installed without being tested.